Cargo.toml

```diff
@@ -59,7 +62,7 @@ sqlite-bundled = ["sqlite", "rusqlite/bundled"]
 compact_filters = ["rocksdb", "socks", "lazy_static", "cc"]
 key-value-db = ["sled"]
 all-keys = ["keys-bip39"]
-keys-bip39 = ["bip39"]
+keys-bip39 = ["pbkdf2", "unicode-normalization", "sha2", "hmac"]
```

Comment:
You should probably have different features for the different languages, as most of the people will use english only anyways

Reply:
Yeah, agreed. I'll add that.

Reply:
Added language feature flags. English is enabled by default.